events-application-rest-api/src/main/java/com/bounswe/bounswe2017group3/Repository/EventRepository.java

```diff
@@ -10,7 +10,10 @@
 public interface EventRepository extends JpaRepository<Event, Long> {
 
     public Event findByName(String name);
-
+    
+    //List events with respect to privacy option
+    public List<Event> findByPrivacyoption(Boolean privacyoption);
```

Comment:
Can you change the name `privacyoption` to `private`? I'd appreciate it since it's more appropriate 😄 Also, if you make this name change, please change the other places too.

Example: Not `localhost:8080/event?privacyoption=false`, it's better if we make it like `localhost:8080/event?private=false`